qutip/tests/test_superoper.py

```diff
@@ -218,14 +218,13 @@ class TestSuper_td:
     A test class for the QuTiP superoperator functions.
     """
 
-    def __init__(self):
-        N = 3
-        self.t1 = QobjEvo([qeye(N)*(1.+0.1j),[create(N)*(1.-0.1j),f]])
-        self.t2 = QobjEvo([destroy(N)*(1.-0.2j)])
-        self.t3 = QobjEvo([[destroy(N)*create(N)*(1.+0.2j),f]])
-        self.q1 = qeye(N)*(1.+0.3j)
-        self.q2 = destroy(N)*(1.-0.3j)
-        self.q3 = destroy(N)*create(N)*(1.+0.4j)
+    N = 3
```

Comment:
It might be nice to initialize these for each test using `setup_method` (https://docs.pytest.org/en/stable/xunit_setup.html#method-and-function-level-setup-teardown) or similar, but having them here is no worse than things were before, although it does hide the problem a bit by avoiding the warning.

Reply:
Yeah, I'll just change the `__init__` into a `setup_method`. That's clearly what was intended by the test anyway.

This was less of a warning and more of a testing failure - pytest refuses to collect tests that have class initialisers.